aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx

```diff
@@ -103,6 +105,27 @@ function Controls(
           }
         />
       </div>
+      <Tooltip
+        title={props.additionalProperties.stacking ? 'Group stacked' : 'Stack'}
+      >
+        <div
+          className={classNames('Controls__anchor', {
+            active: props.additionalProperties.stacking,
+            outlined: props.additionalProperties.stacking,
+            disabled: _.isEmpty(props.orderedMap),
+          })}
+          onClick={
+            !_.isEmpty(props.orderedMap) ? props.onStackingToggle : undefined
```

Comment:
Here you will receive the react warning about the synthetic events please replace undefined to noop or you can have conditional props which can combine onClick prop conditionally.

Reply:
i try reproduce warnings but i can't , but for safety and readability i will change it to noop, thanks

Reply:
tried to reproduce warnings but I can't, but for safety and readability I will change it to `noop`, thanks